css/main.css

```diff
@@ -1,3 +1,96 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+
+body {
+    font-family: 'Quicksand', sans-serif;
+    -webkit-font-smoothing: antialiased;
+    position: relative;
+    color:black;
+    margin-left: 15px;
+    margin-right: 15px;
+    
+}
+
+.appointment-button {
+    padding: 5px;
+    border: none;
+    color: black;
+
+}
+ 
+.appointment-button:hover {
+    background-color: #4D7369;
+    color: white;
+    border: none;
+    transition: all 0.7s ease;
+}
+
+.title {
+    font-family: 'Libre Baskerville', serif;
+}
+
+
+ h1 {
+    font-family: 'Libre Baskerville', serif;
+    font-size: 2em + 5vw;
+    color: black;
+ }
+
+h6 {
+    font-family: 'Libre Baskerville', serif;
+}
+
+
+
+.bg-limegreen {
+    background-color:#E2F6CD;
+}
+.bg-lightgrey {
+    background-color:#F2F2F1;
+}
+
+.bg-lightmint {
+    background-color: #CFF4EC;
+}
+
+.bg-mediummint {
+    background-color:#94D6C8;
+}
+
+.bg-darkgreen {
+    background-color: #4D7369;
+}
+
+.about {
+    min-height: 600px;
+}
+
+.hero {
+    min-height: 400px;
+}
+
+.services {
+    min-height: 500px;
+}
+
+.more-services {
+    margin-top: -150px;
+}
+
+.myDIV {
+    transform:translateY(-200px);
+}
+  
+.social-media-img {
+    width: 1rem;
+}
+
+@media (max-width: 600px) {
+    .more-services {
+        margin-top: 0px;
+    }
+    .logo {
+        text-align: center;
+    }
+  }
```

Comment:
This is a desktop-first approach, meaning your code works for desktop by default, and then your media queries target the mobile size (from `0px` until `600px`). 
It's best practice nowadays to follow a mobile-first approach: you write your code for mobile only, and then add media queries (using **min**-width) to make it work on desktop.